Orange/widgets/utils/webview.py

```diff
@@ -0,0 +1,122 @@
+import textwrap
```

Comment:
Why not replace Orange/widgets/webview.py with this module, and have an additional WebView class that extends one or the other?